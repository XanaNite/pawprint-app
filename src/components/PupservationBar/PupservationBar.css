* {
    box-sizing: border-box;
}

.pupservationBar-container{
    text-align: center;
    border-top: 1px solid lightgray;
    padding: 20px;
    background: #214A77;
}

.pupservationBar-link{
    text-decoration: none;
    font-weight: bolder;
    color: white;
}