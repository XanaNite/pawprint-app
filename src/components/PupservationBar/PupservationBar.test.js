import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PupservationBar from './PupservationBar';

describe(`PupservationBar component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  };

  it('renders PupservationBar by default', () => {
    const wrapper = shallow(<PupservationBar />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('renders the PupservationBar given props', () => {
    const wrapper = shallow(<PupservationBar {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})
