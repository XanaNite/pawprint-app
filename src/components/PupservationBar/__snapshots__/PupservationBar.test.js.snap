// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PupservationBar component renders PupservationBar by default 1`] = `
<div
  className="pupservationBar-container"
>
  <Link
    className="pupservationBar-link"
    to="/pupservations"
  >
    Make a Pupservation
  </Link>
</div>
`;

exports[`PupservationBar component renders the PupservationBar given props 1`] = `
<div
  className="pupservationBar-container"
>
  <Link
    className="pupservationBar-link"
    to="/pupservations"
  >
    Make a Pupservation
  </Link>
</div>
`;
